handler/src/main/java/io/netty/handler/ssl/OpenSslEngine.java

```diff
@@ -976,7 +978,7 @@ public synchronized boolean isOutboundDone() {
     }
 
     @Override
-    public void setEnabledCipherSuites(String[] cipherSuites) {
+    public final void setEnabledCipherSuites(String[] cipherSuites) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Consider using varargs for methods or constructors which take an array the last parameter. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=pmd%3AUseVarargs)